src/NuGet.Services.Metadata.Catalog.Monitoring/Status/PackageMonitoringStatusListItem.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Services.Metadata.Catalog.Helpers;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class PackageMonitoringStatusListItem
+    {
```

Comment:
Why PackageMonitoringStatusListItem and not only PackageMonitoringStatus

Reply:
The actual `PackageMonitoringStatus` object is much more complex. This object is designed to be returned by the `IPackageMonitoringStatusService.ListAsync`, which does not have the full information from the status blobs.